### ISSUE TITLE
Report back on using plain capacitors and TL074

### ISSUE THREAD

**Issue by drChungus** [kstammits/crimps#1]
Heyy,

Believe it or not, I am also considering building the Kinks nowadays and had the same idea to exchange the opamp to a more generic (and currently available) alternative. I was just wondering if you could go ahead with the build with the TL074 and if it really did make a difference in the S&H performance?

Thanks!

**Comment by kstammits** [comment]
I'm having other issues on this version of the board. Will test out opamps after the rest of it is working.   For sure we don't need tantalums, i'm using MLCC and it seems okay. Kept with the C0G on the hold cap, and have so far the wrong JFET.  Today's board layout has some crosstalk problem where input noise can trigger the sampling, so that's broken. needs a re-route.

**Comment by kstammits** [comment]
The strange self-triggering behavior, as it turns out, is the nature of a JFET based analog S&H. The voltage between gate and source needs to be better than 3 or 4 volts to ensure it is not sampling. Initially my noise source, normalled to input, was running +/-9V and whenever it hit -7V it would initiate a sample. This is solved in the original Kinks by changing the noise source to only +/-5V. Same here, I'm adjusting R27 to 1M so we have noise at +/-6V and no more self-triggers.  

 This is relevant to the opamp choice question because the TL074 cannot output within 3-4 volts of its negative rail.  therefore the trigger/not trigger state is about -7V, and your JFET will sample whenever the input gets to about -3V, which is all the darn time with good noise input.  

You can workaround this problem by further reducing the internal noise source at R27 to maybe 330K, but I think that really hobbles the utility.  In this instance we want a better than average opamp so that it can well encompass the plain opamps the rest of your system is feeding it.  If your input is a not rail-to-rail opamp then it never reaches this problematic region. Maybe also good in 15V systems. 

Have not tested cheaper holding capacitors, expect it to reduce the hold time (increase droop), and I don't know what hold time is acceptable for all users, so there's no harm keeping with the special ones at fifteen cents each.    https://www.mouser.com/ProductDetail/810-CGA4C2C0G1H103J

**Comment by KeLaiFu** [comment]
sorry its a little vague, but I saw another version of this posted a few days ago that mentioned some additional tuning was needed for the S&H circuit. I think by tuning they meant carefully matching components.

**Comment by kstammits** [comment]
The key issue with the opamp seems to be the rail-to-rail nature. the sampling/hold action triggers whenever the trigger/gate goes high. The JFET calls high anything not more than 3 or 4 volts below the inputs. You could maybe find a special JFET who thinks that threshold is 2 volts or 5 volts. But in any event, when the trigger is not being triggered, the opamp is buffering it as low as it can go. For a TL074 that might be only -8 volts, so your S&H will take a sample whenever the input gets to -4V; and with a +/-5V noise normalled to in, that's all the time, and the module is useless.  So you could tune the normal noise to just +/-2V, or use a rail-to-rail opamp, so it can reproduce -11V and withstand a -5V at the input. 

In conclusion, the originally specced OPA**** works better than a cheaper opamp.
